recipes/scikit-fem/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "scikit-fem" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "daba60c8ae906877bb6f2b06b09b14b88cd133c6262900dc27b1e40883b72fb9"
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
Same noarch comment

Reply:
this package requires python >= 3.6

Reply:
Also, you can put `python >=3.6` in the requirements below

Reply:
Hi @scopatz sorry I didn't realise you commented here an above. thanks for the tip